test/e2e/network_test.go

```diff
@@ -23,9 +23,12 @@ import (
 
 // testNetwork runs all the cluster and node network tests
 func testNetwork(t *testing.T) {
+	// Populate the global test context
 	testCtx, err := NewTestContext()
 	require.NoError(t, err)
-	require.NoError(t, testCtx.createNamespace(testCtx.workloadNamespace), "error creating test namespace")
+	err = testCtx.waitForWindowsNodes(gc.numberOfNodes, true, false, false)
+	assert.NoError(t, err, "Windows node creation failed")
```

Comment:
no creation happening here, `timed out waiting for Windows nodes` is more proper

Reply:
this comments need to be addressed, LGTM apart from it.